### PR TITLE
Create logger function

### PR DESCRIPTION
**Pull request by Mubangizi**
# Description
Add function `log_activity()` that takes in parameters and saves them to MONGO_DB

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Can This Be Tested?
Add `MONGO_URI` to **ENV** variables and try creating a project, it should add the activity to your Mongo_DB

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by MutegekiHenry — DISMISSED** [review]
Well done @Mubangizi !

File: app/controllers/project.py [hunk 1]
```diff
@@ -18,7 +19,6 @@
 from flask_jwt_extended import jwt_required, get_jwt_identity, get_jwt_claims
 from app.helpers.db_flavor import get_db_flavour
 from app.schemas.monitoring_metrics import BillingMetricsSchema
-from app.misc.mongo_ops import save_document, ACTION_LOG
```
Comment (bashaashma) [inline app/controllers/project.py:21]: I think since you removed this (importing save_document and ACTION_LOG) it gives an error on line 129.

Comment (bashaashma) [reply app/controllers/project.py:21]: since  save_document is being called in line 129

Comment (e-ian) [reply app/controllers/project.py:21]: ditto on this @Mubangizi

Comment (Mubangizi) [reply app/controllers/project.py:21]: Hey @bashaashma  Thank you for this observation

**Comment by e-ian** [comment]
Nice work @Mubangizi , kindly rebase with develop-new to get the latest changes that report build status

**Pull request merged**